packages/graphql/package.json

```diff
@@ -37,13 +38,15 @@
     "graphql-tag": "^2.10.0",
     "react": "^16.4.0",
     "react-apollo": "^2.2.3",
-    "react-dom": "^16.4.0"
+    "react-dom": "^16.4.0",
+    "react-helmet": "^5.2.0"
```

Comment:
Yes, and probably `react-router-dom` too, because these are peer dependencies of `@untool/react` and all peer dependencies that are not being satisfied by this package need to be forwarded.
https://github.com/untool/untool/blob/master/packages/react/package.json#L33

Reply:
This isn't resolved yet - `react-router-dom` is still missing in peer and dev dependencies ;)